### PR TITLE
Add soft link to s3_setup script in /usr/local/bin

### PR DESCRIPTION
**Pull request by amitwac2608**
Signed-off-by: Amit Kumar <amit.kumar@seagate.com>

### CODE REVIEW

**Comment by amitwac2608** [comment]
Jenkins job on latest commit **PASSED**: http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/518

File: rpms/s3/s3rpm.spec [hunk 1]
```diff
@@ -175,15 +175,13 @@ cd %{_builddir}/%{name}-%{version}-%{_s3_git_ver}/scripts/provisioning/s3setup/s
 python%{py_ver} -m compileall -b *.py
 cp *.pyc %{_builddir}/%{name}-%{version}-%{_s3_git_ver}/scripts/provisioning/s3setup/build/lib/s3setup
 
-echo "build complete"
```
Comment (vimalraghwani20) [inline rpms/s3/s3rpm.spec:178]: I have already addressed this in my PR :)

**Review by vimalraghwani20 — APPROVED** [review]

**Review by sachitanands — APPROVED** [review]
How we are checking this softlink is working or not,
is it custom-build and deployment ?

File: rpms/s3/s3rpm.spec (hunk 1 above)
Comment (amitwac2608) [reply rpms/s3/s3rpm.spec:178]: ok, I guess my PR will get merged before yours, so let me do it :)

**Comment by amitwac2608** [comment]
> How we are checking this softlink is working or not,
> is it custom-build and deployment ?

Yes, I deployed custom build on fresh VM.

Tested that soft link gets created on installation, and deleted on package uninstallation.

**Review by AjinkyaDhumal — APPROVED** [review]

File: rpms/s3/s3rpm.spec [hunk 2]
```diff
@@ -441,8 +440,16 @@ sh /opt/seagate/cortx/auth/scripts/swupdate/merge.sh
...
+ln -sf /opt/seagate/cortx/s3/bin/s3_setup /usr/local/bin/s3_setup
+chmod 755 /usr/local/bin/s3_setup
+
 openssl_version=`rpm -q --queryformat '%{VERSION}' openssl`
 if [ "$openssl_version" != "1.0.2k" ] && [ "$openssl_version" != "1.1.1" ]; then
   echo "Warning: Unsupported (untested) openssl version [$openssl_version] is installed which may work."
   echo "Supported openssl versions are [1.0.2k, 1.1.1]"
 fi
+
+%postun
```
Comment (sachitanands) [inline rpms/s3/s3rpm.spec:454]: spelling mistake .. may be "%postrun"

**Comment by amitwac2608** [comment]
RPM creation jenkins job passed: http://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/centos-7.8/job/s3-custom-build/2486

File: rpms/s3/s3rpm.spec (hunk 2 above)
Comment (amitwac2608) [reply rpms/s3/s3rpm.spec:454]: I checked, its %postun.

Comment (sachitanands) [reply rpms/s3/s3rpm.spec:454]: yes. reference link : https://rpm-packaging-guide.github.io/#triggers-and-scriptlets

**Comment by gowthamchinna** [comment]
run premerge-tes

**Comment by gowthamchinna** [comment]
run premerge-test

**Pull request merged**